peekingduck/pipeline/nodes/input/visual.py

```diff
@@ -122,20 +126,22 @@ class Node(AbstractNode):  # pylint: disable=too-many-instance-attributes
     Ok : normal behavior |br|
     \+ : potentially faster FPS |br|
     ! : lost frames if source is faster than PeekingDuck |br|
-    !! : laggy video, potential out-of-memory error due to buffer overflow if source
-    is faster than PeekingDuck
+    !! : "slow-mo" video, potential out-of-memory error due to buffer overflow
+    if source is faster than PeekingDuck
 
     Note: If threading=False, then the secondary parameter buffering is ignored
     regardless if it is set to True/False.
 
     Here is a video to illustrate the differences between
-    `a normal video vs a laggy video
+    `a normal video vs a "slow-mo" video
     <https://storage.googleapis.com/peekingduck/videos/wave_normal_vs_laggy.mp4>`_
-    using a 30 FPS webcam: the video on the right lags the normal video on the left.
-    This happens as both threading and buffering are set to True, and the threaded
-    ``input.visual`` reads the webcam at almost 60 FPS.
-    Since the hardware is physically limited at 30 FPS, this means every frame gets duplicated,
-    resulting in each frame being processed and shown twice, thus "stretching out" the video.
+    using a 30 FPS webcam: the video on the right appears to be playing in slow
+    motion compared to the normal video on the left.
+    This happens as both threading and buffering are set to True, and the
+    threaded :mod:`input.visual` reads the webcam at almost 60 FPS.
+    Since the hardware is physically limited at 30 FPS, this means every frame
+    gets duplicated, resulting in each frame being processed and shown twice,
+    thus "stretching out" the video.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
```

Comment:
Suggest removing all the comments and technotes from `visual.py` as they have been explained in RTD.

Reply:
Removed